app/main.py

```diff
@@ -24,6 +26,29 @@
     allow_credentials=True,
 )
 
+# Just going to create a dictionary for now. Not sure how it would fit into current database.
+# Probably should add this to the database, but that might need further discussion.
+# key is first letter of postal code, value is province_code in db.
+postal_to_province = {
```

Comment:
yeah we can just put this in the db later so it is consistent and the keys update if they change, but this is good for now